orangecontrib/text/widgets/owsentimentanalysis.py

```diff
@@ -22,40 +26,85 @@ class Outputs:
 
     method_idx = settings.Setting(1)
     autocommit = settings.Setting(True)
-    language = settings.Setting('English')
+    liu_language = settings.Setting('English')
+    multi_language = settings.Setting('English')
     want_main_area = False
     resizing_enabled = False
 
     METHODS = [
-        Liu_Hu_Sentiment,
-        Vader_Sentiment
+        LiuHuSentiment,
+        VaderSentiment,
+        MultiSentiment
     ]
     LANG = ['English', 'Slovenian']
+    MULTI_LANG = MultiSentiment.LANGS.keys()
+
+    class Warning(OWWidget.Warning):
+        senti_offline = Msg('No internet connection! Sentiment now only works '
+                            'with local models.')
+        senti_offline_no_lang = Msg('No internet connection and no local '
+                                    'language resources are available.')
 
     def __init__(self):
         super().__init__()
         self.corpus = None
 
-        form = QGridLayout()
+        self.form = QGridLayout()
         self.method_box = box = gui.radioButtonsInBox(
             self.controlArea, self, "method_idx", [], box="Method",
-            orientation=form, callback=self._method_changed)
+            orientation=self.form, callback=self._method_changed)
         self.liu_hu = gui.appendRadioButton(box, "Liu Hu", addToLayout=False)
-        self.liu_lang = gui.comboBox(None, self, 'language',
+        self.liu_lang = gui.comboBox(None, self, 'liu_language',
```

Comment:
Use `contentsLength` parameter for bouth `gui.comboBox` so that they are of equal length.